nmp/all.nmp

```diff
@@ -2162,13 +2114,12 @@ op pwrite_xar7(src: LOC16)
 	}
 
 // DOC: page 385
-// NP: one op, only one bit changes
-// op 
-// 	syntax = 
-// 	image = 
-// 	action = {
+op qmacl(optype: bit, loc: LOC32)
+	syntax = if optype == 0b1 then format("QMACL P, %s, *XAR7", loc) else format("QMACL P, %s, *XAR7++", loc) endif
```

Comment:
You can simply `format("... %s, %s", if optype == 1 then "*XAR7++" else "*XAR7", ...)`

Reply:
Convenient, added!